docs/docs/manual/exporting.md

```diff
@@ -6,34 +6,97 @@ sidebar_label: Exporting
+
+With the download options, you can generate a preview of how the first ten rows of your dataset will output. If you do not choose one of the file formats on the right, the "Download" button will generate a text file. 
+
+With the <span class="menuItems">Option Code</span> tab, you can copy JSON data that saves your current settings, to reuse on another project, or you can paste in existing JSON settings data to apply to the current project. 
+
+### SQL exporter
+
+The SQL exporter creates a SQL statement containing the data you’ve exported, which you can use to overwrite or add to an existing database. 
+
+![A screenshot of the SQL statement creator window.](/img/sql-exporter.png)
+
+A window with two tabs will pop up: one to define what data to output, and another to modify the statement table. This allows you to craft a complete SQL command, including DROP and IF EXISTS if you require them. 
+
+You can set a default value for empty cells after unchecking “Allow null” in one or more columns. Trimming column names will remove their whitespace characters. 
+
+![A screenshot of the SQL statement download window.](/img/sql-exporter2.png)
+
+“Include schema” means that you will start your statement with the creation of a table. Without that, you will only have an INSERT statement. 
+“Include content” means the INSERT statement with data from your project. Without that, you will only create empty columns. 
+
+You can preview the statement, which will open up a new browser tab/window showing a statement with the first ten rows of your data, or you can save a `.sql` file to your computer. 
+
+### Templating exporter
+
+If you pick <span class="menuItems">Templating…</span> from the <span class="menuItems">Export</span> dropdown menu, you can “roll your own” exporter. This is useful for formats that we don't support natively yet, or won't support. The Templating exporter generates JSON by default. 
+
+The window that appears allows you to set your own separators, prefix, and suffix to create a complete dataset in the language of your choice. In the <span class="menuItems">Row Template</span> section, you can choose which columns to generate from each row by calling them with variables. 
+
+This can be used to:
+* output reconciliation data (`cells["column name"].recon.match.name`, `.recon.match.id`, and `.recon.best.name`, for example) instead of cell values
+* create multiple columns of output from different member fields of a single project column
+* employ GREL expressions to modify cell data for output (for example, `cells["column name"].value.toUppercase()`). 
+
+Anything that appears inside curly braces ({{}}) is treated as a GREL expression; anything outside is generated as straight text. You can include [regular expressions](expressions#regular-expressions) as usual (inside forward slashes, with any GREL function that accepts them), with the exception that you cannot use curly braces (for example, to denote a `{min,max}` capture range). 
```

Comment:
I think the inclusion of curly braces in the templates is more general than just the instance of RegEx in GREL. Need to check this but we may need a more blanket statement about this